src/main/java/com/verapdf/restapi/entity/Job.java

```diff
@@ -0,0 +1,65 @@
+package com.verapdf.restapi.entity;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.Closeable;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.UUID;
+
+public class Job implements Closeable {
+
+    private static final Logger log = LoggerFactory.getLogger(Job.class);
+
+    private UUID jobId;
+
+    private List<File> files;
+    private List<File> tempFiles;
+
+    public Job() {
+        files = new LinkedList<>();
+        tempFiles = new LinkedList<>();
+        setJobId(UUID.randomUUID());
+    }
+
+    public UUID getJobId() {
+        return jobId;
+    }
+
+    private void setJobId(UUID jobId) {
+        this.jobId = jobId;
```

Comment:
remove this method. when some job has been created we don't want to change its id